### PR TITLE
chore: Replaced .length > 0 with .!empty?

### LINKED ISSUE #32: Replace .length > 0 with !empty?
#### Issue Summary

According to [CodeClimate](https://codeclimate.com/github/sendgrid/smtpapi-ruby/pull/31) we are using .length > 0 in a number of places, please go through and replace them all! 

#### Acceptance
There may be other issues reported by CodeClimate, but there shouldn't be any more .length > 0  issues. The other issues on this report will be added to another issue.

### PR DESCRIPTION
**Pull request by nvzard**
# Fixes https://github.com/sendgrid/smtpapi-ruby/issues/32

### Checklist
- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [ ] I have read the [Contribution Guide] and my PR follows them.
- [ ] I updated my branch with the master branch.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation about the functionality in the appropriate .md file
- [ ] I have added in line documentation to the code I modified

### Short description of what this PR does:
- 
- 

If you have questions, please send an email to [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/smtpapi-ruby?pullRequest=46) <br/>All committers have signed the CLA.

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/smtpapi-ruby?pullRequest=46) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/smtpapi-ruby?pullRequest=46) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/smtpapi-ruby?pullRequest=46) it.</sub>

**Comment by nvzard** [comment]
@thinkingserious please review.

File: lib/smtpapi.rb [hunk 1]
```diff
@@ -116,20 +116,20 @@ def set_ip_pool(pool_name)
 
     def to_array
       data = {}
-      data['to'] = @to if @to.length > 0
```
Comment (mbernier) [inline lib/smtpapi.rb]: It looks like the Travis build is having some issues with the .emtpy? calls here: https://travis-ci.org/sendgrid/smtpapi-ruby/builds/294550416#L652

Can you please look into this and see what's causing these errors?

**Review by mbernier — CHANGES_REQUESTED** [review]

**Review by mbernier — APPROVED** [review]

**Comment by thinkingserious** [comment]
Hello @nvzard,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**